src/bun.js/api/server/SSLConfig.zig

```diff
@@ -227,6 +227,89 @@ pub fn deinit(this: *SSLConfig) void {
         this.ca = null;
     }
 }
+pub fn clone(this: *const SSLConfig) SSLConfig {
+    var cloned: SSLConfig = .{
+        .secure_options = this.secure_options,
+        .request_cert = this.request_cert,
+        .reject_unauthorized = this.reject_unauthorized,
+        .client_renegotiation_limit = this.client_renegotiation_limit,
+        .client_renegotiation_window = this.client_renegotiation_window,
+        .requires_custom_request_ctx = this.requires_custom_request_ctx,
+        .is_using_default_ciphers = this.is_using_default_ciphers,
+        .low_memory_mode = this.low_memory_mode,
+        .protos_len = this.protos_len,
+    };
+    const fields = .{
+        "server_name",
+        "key_file_name",
+        "cert_file_name",
+        "ca_file_name",
+        "dh_params_file_name",
+        "passphrase",
+        "protos",
+    };
+
+    inline for (fields) |field| {
+        if (@field(this, field)) |slice_ptr| {
+            const slice = std.mem.span(slice_ptr);
+            @field(cloned, field) = bun.handleOom(bun.default_allocator.dupeZ(u8, slice));
+        }
+    }
+
+    if (!this.is_using_default_ciphers) {
+        if (this.ssl_ciphers) |slice_ptr| {
+            const slice = std.mem.span(slice_ptr);
+            if (slice.len > 0) {
+                cloned.ssl_ciphers = bun.handleOom(bun.default_allocator.dupeZ(u8, slice));
+            } else {
+                cloned.ssl_ciphers = null;
```

Comment:
```suggestion
            cloned.ssl_ciphers = if (slice.len > 0) bun.handleOom(bun.default_allocator.dupeZ(u8, slice)) else null;
```

This is an expression, which can be stored into an immutable, contrasted with a set of statements, which cannot.

https://stackoverflow.com/questions/4192225/benefits-of-ternary-operator-vs-if-statement

Reply:
I got you

```zig
cloned.ssl_ciphers =
    if (slice.len > 0)
        bun.handleOom(bun.default_allocator.dupeZ(u8, slice))
    else
        null;
```